packages/astro/src/core/build/generate.ts

```diff
@@ -192,14 +192,16 @@ export async function generatePages(options: StaticBuildOptions, internals: Buil
 	await runHookBuildGenerated({ config, logger });
 }
 
+const THRESHOLD_SLOW_RENDER_TIME_MS = 500;
```

Comment:
is this number based on any data? I'm not really sure what to expect the value to be.

Reply:
No, it isn't based on any data.

Reply:
Is there a way that we can collect some data to have a better understanding what the expected render time is before deploying this kind of change? My only worry is that 500 is actually not that long and users start reporting that they are getting what they think is an error in their console.

Reply:
Given outside discussions no issue with going with this number for now. We should adjust it if it turns out a lot of people are receiving false-positives on this, however.